src/agent/definitions/toolrunner.d.ts

```diff
@@ -20,6 +20,6 @@ export declare class ToolRunner extends events.EventEmitter {
     args: string[];
     silent: boolean;
     private _debug(message);
-    arg(arguments: any, raw: any): void;
```

Comment:
vso-task-lib repo is generating the definition files now.  are we checking in those?